pkg/cli/admin/prune/images/images.go

```diff
@@ -470,28 +473,26 @@ func (o PruneImagesOptions) Run() error {
 	}
 
 	options := imageprune.PrunerOptions{
-		KeepYoungerThan:       o.KeepYoungerThan,
-		KeepTagRevisions:      o.KeepTagRevisions,
-		PruneOverSizeLimit:    o.PruneOverSizeLimit,
-		AllImages:             o.AllImages,
-		Images:                allImages,
-		ImageWatcher:          imageWatcher,
-		Streams:               allStreams,
-		StreamWatcher:         imageStreamWatcher,
-		Pods:                  allPods,
-		RCs:                   allRCs,
-		BCs:                   allBCs,
-		Builds:                allBuilds,
-		DSs:                   allDSs,
-		Deployments:           allDeployments,
-		DCs:                   allDCs,
-		RSs:                   allRSs,
-		LimitRanges:           limitRangesMap,
-		DryRun:                o.Confirm == false,
-		RegistryClientFactory: registryClientFactory,
-		RegistryURL:           registryURL,
-		PruneRegistry:         o.PruneRegistry,
-		IgnoreInvalidRefs:     o.IgnoreInvalidRefs,
+		KeepYoungerThan:    o.KeepYoungerThan,
+		KeepTagRevisions:   o.KeepTagRevisions,
+		PruneOverSizeLimit: o.PruneOverSizeLimit,
+		AllImages:          o.AllImages,
+		Images:             allImages,
+		ImageWatcher:       imageWatcher,
```

Comment:
Where are we leveraging this ? (I could not find)

Reply:
we don't, it was used in 3.x, but it was disabled because of bugs
removed, but we'll need to restore it eventually